code/modules/mob/living/carbon/human/species/moth.dm

```diff
@@ -1,6 +1,7 @@
 #define COCOON_WEAVE_DELAY 5 SECONDS
 #define COCOON_EMERGE_DELAY 15 SECONDS
 #define COCOON_HARM_AMOUNT 50
+#define COCOON_NUTRITION_REQUIREMENT 201
 #define COCOON_NUTRITION_AMOUNT -200
```

Comment:
put a negative infront of where the `COCOON_NUTRITION_AMOUNT` and then reuse this define.